src/rules/noDuplicateKeyRule.ts

```diff
@@ -31,6 +31,7 @@ export class Rule extends Lint.Rules.AbstractRule {
         options: null,
         optionExamples: ["true"],
         type: "functionality",
+        typescriptOnly: false,
```

Comment:
This rule was deleted on the master branch because it is covered by the compiler. I'm going to delete it again; I don't want to maintain it _just_ for JS files.